geo-types/src/geometry.rs

```diff
@@ -207,6 +114,7 @@ macro_rules! try_from_geometry_impl {
     }
 }
 
+// `concat_idents` is not available, so hacking around it
```

Comment:
I don't understand what this comment is about. Are you explaining the trailing comma?